v8_c_api/src/v8_c_api.cpp

```diff
@@ -476,14 +510,22 @@ void v8_FreeContextRef(v8_context_ref *v8_ctx_ref) {
 }
 
 void* v8_GetPrivateDataFromCtxRef(v8_context_ref* ctx_ref, size_t index) {
-	v8::Local<v8::External> data = v8::Local<v8::External>::Cast(ctx_ref->context->GetEmbedderData(index + 2));
```

Comment:
Same question as above, I see we did not handle it before as well but maybe we can handle it now?